tools/pycozmo_app.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+"""
+
+PyCozmo application.
+
+"""
+
+import sys
+import time
+import argparse
+
+import pycozmo
+
+
+def parse_args():
+    """ Parse command-line arguments. """
+    parser = argparse.ArgumentParser(description=__doc__)
+    args = parser.parse_args()
+    return args
+
+
+def brain(cli: pycozmo.client.Client):
+    """ Perform robot OTA firmware update. """
```

Comment:
Do we need a firmware update here? :wink:

Reply:
Oops.